tests/conftest.py

```diff
@@ -0,0 +1,46 @@
+import os
+import json
+import sys
+from pytest import fixture
+
+
+def json_from_file(request, filename):
```

Comment:
nice.  wish this one was built in...

Reply:
I think I may have found a pytest lib to manage test data at some point, but for so few files this works well enough. I do wish it was included in pytest though...